lib/cache/cache.go

```diff
@@ -708,7 +708,7 @@ func (c *Cache) update(ctx context.Context, retry utils.Retry) {
 			return
 		}
 		if err != nil {
-			c.WithError(err).Warning("Re-init the cache on error.")
+			c.Warningf("Re-init the cache on error: %v.", err)
```

Comment:
nit: Does the`c.WithError(err).Warning` is not recommended style ?

Reply:
Yes, it is. However, in some cases it may produce overly verbose log entries especially when the logging occurs within a loop and/or the backtrace isn't likely to be very informative -- both of which is the case here.